Moderation-Policy.md

```diff
@@ -0,0 +1,48 @@
+## Moderation policy
+
+### Applicability
+
+By default, this policy applies to all repositories under the Node.js Github Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
+* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
+* Includes provisions for clearly and openly documenting Moderation actions taken.
+
+### Terms
+
+* "Collaborator" refers to any individual member or "outside collaborator" of the Node.js Github Organization.
```

Comment:
If I am not mistaken this is dealt with in line 21

--> https://github.com/nodejs/TSC/pull/16/files#diff-ba1bbe10e98d34ecb91c62040189c025R21